processor/src/chiplets/bus/mod.rs

```diff
@@ -7,6 +7,8 @@ use crate::Matrix;
 mod aux_trace;
 pub use aux_trace::AuxTraceBuilder;
 
+type Cycle = u32;
+
```

Comment:
I agree that this improves readability. However, I think I'd prefer not to make this change for the following reasons:

1. We usually refer to this as the clock cycle elsewhere, so I think this ends up being a bit inconsistent with other parts of the codebase where we use `clk` a lot.
2. we use clock cycles all over the place int he VM, so if we're making this change I'd prefer to make it everywhere rather than within this one module.

Reply:
> we use clock cycles all over the place int he VM, so if we're making this change I'd prefer to make it everywhere rather than within this one module.

I usually try to do small incremental changes, instead of the whole codebase unless that is done by a tool. Because the PRs get super large with mechanical work, that can be easy to miss some steps on implementation and review.

Reply:
Yeah, that makes sense, but I wouldn't want to make a change like this in one place unless we decide first that we're going to make it everywhere, which I'm not sure we want to do. If we did break it down into incremental changes, I would probably prefer those to be in their own PRs fairly close together and not mixed in with other things.